File: modules/01-basic-syntax/python/slices.py
print("------------------------\n")
# Array slices
print("Array slices")

my_ints_array = [10, 20, 30, 40, 50, 60]
print(f"my_ints_array: {my_ints_array}")

my_ints_slice = my_ints_array[2:4]
print(f"my_ints_slice: {my_ints_slice}")

print("------------------------\n")
# String vs &str
print("String vs &str")

s1 = "World"
print(f"s1: {s1}")

s2 = "Hello "
print(f"s2: {s2}")
s2 += s1
print(f"s2: {s2}")

s3 = s2[6:]
print(f"s3: {s3}")
print("NOTE: Python does not have different types of string")

File: modules/01-basic-syntax/typescript/slices.ts
console.log("------------------------\n");
// Array slices
console.log('Array slices');

const my_ints_array = [10, 20, 30, 40, 50, 60];
console.log(`my_ints_array: ${my_ints_array}`);

const my_ints_slice = my_ints_array.slice(2,4);
console.log(`my_ints_slice: ${my_ints_slice}`);

console.log("------------------------\n");
// String vs &str
console.log('String vs &str');

const s1 = 'World';
console.log(`s1: ${s1}`);

let s2 = 'Hello ';
console.log(`s2: ${s2}`);
s2 += s1;
console.log(`s2: ${s2}`);

const s3 = s2.slice(6);
console.log(`s3: ${s3}`);
console.log('NOTE: Python does not have different types of string');
